internal/connectors/plugins/public/tink/payments.go

```diff
@@ -14,6 +15,22 @@ type paymentsState struct {
 	NextPageToken string `json:"nextPageToken"`
 }
 
+func computeCreatedAt(transaction client.Transaction) time.Time {
+	if !transaction.TransactionDateTime.IsZero() {
+		return transaction.TransactionDateTime
+	}
+	if !transaction.Dates.Transaction.IsZero() {
+		return transaction.Dates.Transaction
+	}
+	if !transaction.BookedDateTime.IsZero() {
```

Comment:
Are we sure we want to use the createdAt instead of the bookedAt?

I'm pretty sure I'm using booked for balances for example.

... Do we need to go back to look at all connectors to see what we're actually doing? 😂

Reply:
Also we don't care about value date time? (I don't really know if we care about it at all, I'm not sure what it's meant to be in terms of business domain)